### ISSUE TITLE
Determining what our security policy should be for our repository

### ISSUE THREAD

**Issue by francisfuzz** [copilot-extensions/preview-sdk.js#79]
## Summary

[Open Source Guides](https://opensource.guide/) shares a number of recommended community standards, including adding a [security policy](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository).

I think it would be great if we can consider adding a real `SECURITY.md` file that gives people instructions for reporting security vulnerabilities in our project, if/when ever they should come up.

---

cc: @gr2m for triage 🎫

**Comment by gr2m** [comment]
I agree, we should do that. I have no strong opinion on the process. Do you have a preference? 
GitHub has a process for that now https://github.com/copilot-extensions/preview-sdk.js/security/policy

**Comment by francisfuzz** [comment]
@gr2m - No hard preference. That will open up a new template to create a `SECURITY.md` but the actual details are left up to us. 😅 

I wonder if there's another repo in the `github` org that has reusable content for us. 💭

**Comment by gr2m** [comment]
Maybe we can utilize the "Private vulnerability reporting" feature, and link to that from SECURITY.md?

**Comment by francisfuzz** [comment]
> Maybe we can utilize the "Private vulnerability reporting" feature, and link to that from SECURITY.md?

@gr2m Makes sense! Reading [the docs](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability):

> Owners and administrators of public repositories can enable private vulnerability reporting on their repositories.

I don't have those privileges; if/when you choose to enable it, I can link to it from the `SECURITY.md` (along with said GitHub Docs reference so folks can read more). 👍
